app/models/hbx_enrollment.rb

```diff
@@ -1877,6 +1877,13 @@ def covered_members_first_names
     end
   end
 
+  def covered_members_name_age
+    enrollment_members = hbx_enrollment_members.sort_by { |a| a.is_subscriber ? 0 : 1 }
+    enrollment_members.inject([]) do |name_age, member|
+      name_age << "#{member.person.first_name} (#{((Time.zone.now - member.person.dob.to_time) / 1.year.seconds).floor})"
```

Comment:
is this formmated for screen ? if yes, could this be on a view helper ? if no, would it be more reusable to return it on a hash ?

Reply:
moved to families_helper